SpecFlow_BrowserStack/resources/features/SampleTest.feature

```diff
@@ -1,6 +1,6 @@
 @sample-test
 Feature: Sample
-	Scenario Outline: Can add product to cart
+	Scenario Outline: BStack Sample Test - Can add product to cart
```

Comment:
If we do feature plus Scenario Outline then change Feature name to BStack Sample and BStack Local

Reply:
changed